docs/ballerina-by-example/examples/http-client-connector/http-client-connector.bal

```diff
@@ -59,7 +59,7 @@ function main (string[] args) {
                 json jsonPayload1 => {
                     io:println(jsonPayload1);
                 }
-                mime:EntityError payloadError1 => {
+                http:PayloadError payloadError1 => {
```

Comment:
Why do we say payloadError1 ? Is it because payloadError is already there ?

Reply:
Fixed it